frontend/src/pages/TermPlanner/OptionsHeader/OptionsHeader.tsx

```diff
@@ -71,6 +73,22 @@ const OptionsHeader = ({ plannerRef }: Props) => {
             </Tooltip>
           </div>
         </Tippy>
+        <Tippy
```

Comment:
When clicking the save button, it show any indication that it is being saved to the backend? Also, is there a reason to use tippy when we are not displaying a popup?

Think this could be improved by either of these approaches:
- when clicking the save button, it brings a popup to confirm to save to the backend? (similar to export planner/unschedule all courses buttons) - right now the popup is empty
- when clicking the saving icon, it creates a spinner animation once the save request has been processed and displays a tick icon to indicate it has been save for a few seconds